### ISSUE TITLE
Missing path from XCFramework 'MuppetIOS.xcframework' as defined by 'DebugSymbolsPath' in its `Info.plist` file

### ISSUE THREAD

**Issue by jmccance** [atomicfi/atomic-transact-ios#4]
Seeing a build error when trying to use versions of the SDK above 3.3.4. I see this issue starting from 3.4.2 (the next version listed on the release page), going through to the most recent version (3.5.14).

```
error: Missing path (/Users/jmccance/Library/Developer/Xcode/DerivedData/AtomicDependencyErrorDemo-fgxjuoivsidlhwgwbzhlvsvfzfsx/SourcePackages/checkouts/atomic-transact-ios/artifacts/MuppetIOS.xcframework/ios-arm64_x86_64-simulator/dSYMs)
from XCFramework 'MuppetIOS.xcframework' as defined by 'DebugSymbolsPath' in its `Info.plist` file (in target
'AtomicTransact' from project 'AtomicSDK')
```

I'm able to reproduce this by creating a fresh Xcode project and adding the dependency via project editor.

If there's anything else I can provide to help you track this time, please let me know!

![image](https://github.com/user-attachments/assets/9983a761-02f9-4466-9307-46df2155d5aa)

![image](https://github.com/user-attachments/assets/8eccae14-d044-4d96-a485-286d3ba44788)

**Comment by sean-hill** [comment]
@jmccance thank you for reporting this! We are aware and are working on a solution right now. Thanks for your patience!

**Comment by sean-hill** [comment]
Hey @jmccance this has been resolved with version [3.5.15](https://github.com/atomicfi/atomic-transact-ios/releases/tag/3.5.15) of our SDK, please give it a try! Thank.
